src/handle-data.js

```diff
@@ -5,6 +5,21 @@ const directionMap = {
 
 const MAX_CELLS = 10000;
 
+/**
+ * Get total cell info
+ * @param {Boolean} isDim
+ * @param {Object} layout
+ * @param {Number} colIndex
+ * @param {Number} numDims
+ * @returns dimensions and measures total cell values as strings
+ */
+export function getTotalInfo(isDim, layout, colIndex, numDims) {
+  if (!isDim) return layout.qHyperCube.qGrandTotalRow[colIndex - numDims]?.qText;
+  if (colIndex !== 0) return '\u00A0';
+  if (layout.totals.label !== undefined && layout.totals.label) return layout.totals.label;
```

Comment:
you are checking if it is not undefined then checking if it exists at all? one should be enough